### PR TITLE
sc localserver regression fixes

### PR DESCRIPTION
**Pull request by PepeBotella25**

### CODE REVIEW

File: extensions/localserver-cli-command/src/compilers/Compiler.js [hunk 1]
```diff
@@ -46,6 +46,11 @@ module.exports = class Compiler {
 
 	_createRequireJSFile(localFolder) {
 		const src = require.resolve('requirejs');
-		fs.copyFileSync(src, path.join(localFolder, 'require.js'));
+		const dst = path.join(localFolder, 'require.js');
+		fs.copyFileSync(src, dst);
+
+		let content = fs.readFileSync(dst).toString();
+		content = content.replace('#!/usr/bin/env node', '');
+		fs.writeFileSync(dst, content);
```
Comment (jbbalbas) [inline extensions/localserver-cli-command/src/compilers/Compiler.js]: Instead copying the file from src to dst, then reading it from dst, remove the `#!/usr/bin/env node` string and then write again in dst it, wouldn't it be more efficient to read it from src, remove the unwanted string and write it in dst?

File: extensions/localserver-cli-command/src/DeployXml.js [hunk 2]
```diff
@@ -16,12 +16,20 @@ module.exports = class DeployXml {
...
+		const isACP = manifest.$.projecttype === 'ACCOUNTCUSTOMIZATION';
+
 		let objectsPath = this.deploy.objects.path;
 		objectsPath = path.join(this._projectFolder, objectsPath.substr(1));
 		this.objectsPath = path.dirname(objectsPath);
 
 		let filesPath = this.deploy.files.path;
 		filesPath = path.join(this._projectFolder, filesPath.substr(1));
+
+		if(isACP){
```
Comment (vic-nik) [inline extensions/localserver-cli-command/src/DeployXml.js]: Formatting, spaces are missing after if and before curly brace

File: extensions/localserver-cli-command/src/compilers/Compiler.js (hunk 1 above)
Comment (vic-nik) [reply extensions/localserver-cli-command/src/compilers/Compiler.js]: +1

File: extensions/localserver-cli-command/src/DeployXml.js [hunk 3]
```diff
@@ -16,12 +16,20 @@ module.exports = class DeployXml {
 
 		this.deploy = Utils.parseXml(this._projectFolder, 'deploy.xml').deploy;
 
+		const { manifest } = Utils.parseXml(this._projectFolder, 'manifest.xml');
```
Comment (vic-nik) [inline extensions/localserver-cli-command/src/DeployXml.js]: We have constants for manifest.xml and ACCONTCUSTOMIZATION in ApplicationConstants.js
I would use those here and we can add another constrant for 'deploy.xml' under FILES

**Pull request merged**